Gemfile

```diff
@@ -79,6 +79,7 @@ group :production, :staging do
   gem 'memcachier'
   gem 'rails_12factor' # supresses heroku plugin injection
   gem 'bonsai-elasticsearch-rails'  # Integration with Bonsa-Elasticsearch on heroku
+  gem 'sparkpost_rails'
```

Comment:
gahhh that came in with the merge from the rubocop.yml -editing branch which i apparently made off of a wrong branch or something. *sigh* (that was part of experimenting with using SP's API instead of just using SMTP)